src/cosmoz-dropdown.ts

```diff
@@ -43,6 +45,21 @@ interface DropdownProps
 		Pick<ContentProps, 'placement' | 'render'> {}
 
 const Dropdown = (host: HTMLElement & DropdownProps) => {
+	const supportsPopover = () => {
+		// eslint-disable-next-line no-prototype-builtins
+		return HTMLElement.prototype.hasOwnProperty('popover');
+	};
+
+	const popoverChanged = (popover?: Element) => {
```

Comment:
Move this outside Content and rename it to showPopover.
 Do not even request the animation frame if popover is not supported.